react_frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "dev": "next dev",
+    "dev": "NODE_OPTIONS='--max-http-header-size=2097152' next dev",
```

Comment:
where did this specific size come from?

Reply:
It was quite the journey to get to that number too. So many twists and turns and jump scares.

Reply:
Glad you guys did it together. It's dangerous to go alone!